src/main/resources/db/migration/V1_4__new_schema.sql

```diff
@@ -0,0 +1,188 @@
+values ('Webhook Consumer');
+
+insert into event_consumer (consumer_name)
+values ('Internal Adaptor');
+
+create table consumer_subscription
+(
+    id                   SERIAL PRIMARY KEY       NOT NULL,
+    consumer_id          BIGINT                   NOT NULL,
+    event_type_id        VARCHAR(50)              NOT NULL,
+    poll_client_id       VARCHAR(50),
+    callback_client_id   VARCHAR(50),
+    last_poll_event_time timestamp with time zone,
+    push_uri             varchar(400),
+    nino_required        boolean                  not null default false,
+    when_created         timestamp with time zone not null default now(),
+
+    CONSTRAINT fk_event_type
+        FOREIGN KEY (event_type_id)
+            REFERENCES event_type (id),
+
+    CONSTRAINT fk_consumer
+        FOREIGN KEY (consumer_id)
+            REFERENCES event_consumer (id),
+
+    CONSTRAINT uk_event_client UNIQUE (event_type_id, callback_client_id)
+);
+
+insert into consumer_subscription (poll_client_id, callback_client_id, consumer_id, event_type_id, nino_required)
+values ('dwp-event-receiver', 'dwp-event-receiver', (select id from event_consumer WHERE consumer_name = 'DWP Poller'),
+        'DEATH_NOTIFICATION', true);
+insert into consumer_subscription (poll_client_id, callback_client_id, consumer_id, event_type_id)
+values ('dwp-event-receiver', 'dwp-event-receiver', (select id from event_consumer WHERE consumer_name = 'DWP Poller'),
+        'LIFE_EVENT');
+
+insert into consumer_subscription (callback_client_id, consumer_id, event_type_id, nino_required)
+values ('hmrc-client', (select id from event_consumer WHERE consumer_name = 'Pub/Sub Consumer'),
+        'DEATH_NOTIFICATION', true);
+insert into consumer_subscription (callback_client_id, consumer_id, event_type_id)
+values ('hmrc-client', (select id from event_consumer WHERE consumer_name = 'Pub/Sub Consumer'),
+        'LIFE_EVENT');
+
+insert into consumer_subscription (callback_client_id, consumer_id, event_type_id)
+values ('internal-outbound', (select id from event_consumer WHERE consumer_name = 'Internal Adaptor'),
+        'LIFE_EVENT');
+insert into consumer_subscription (callback_client_id, consumer_id, event_type_id, nino_required)
+values ('internal-outbound', (select id from event_consumer WHERE consumer_name = 'Internal Adaptor'),
+        'DEATH_NOTIFICATION', true);
+
+insert into consumer_subscription (consumer_id, event_type_id, push_uri, nino_required)
+values ((select id from event_consumer WHERE consumer_name = 'S3 Consumer'),
+        'DEATH_NOTIFICATION',
+        's3://user:password@localhost', true);
+
+insert into consumer_subscription (consumer_id, event_type_id, push_uri)
+values ((select id from event_consumer WHERE consumer_name = 'S3 Consumer'),
+        'LIFE_EVENT',
+        's3://user:password@localhost');
+
+insert into consumer_subscription (consumer_id, event_type_id, push_uri)
+values ((select id from event_consumer WHERE consumer_name = 'Webhook Consumer'),
```

Comment:
I'm mildly uncomfortable with this set of data potentially ending up in production/non-local dev locations, can we split it out as a dev script/something generated through infra rather than through the migrations?

Reply:
Agreed